docs/source/notebooks/AR.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<ul><li>Since it's phi in the latex above, it would be best to call the parameter <code>phi</code> instead of <code>beta</code></li><li>Is <code>tau</code> defined somewhere? It won't run otherwise. Or maybe you can infer <code>tau</code>, as you did in the AR1 model?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>